src/Certify.Models/Models/Config/ChallengeProviders.cs

```diff
@@ -41,7 +42,8 @@ public class ChallengeProviders
                     new ProviderParameter{Key="clientid", Name="Application Id", IsRequired=false },
                     new ProviderParameter{Key="secret",Name="Svc Principal Secret", IsRequired=true , IsPassword=true},
                     new ProviderParameter{Key="subscriptionid",Name="DNS Subscription Id", IsRequired=true , IsPassword=false},
-                    new ProviderParameter{Key="resourcegroupname",Name="Resource Group Name", IsRequired=true , IsPassword=false}
+                    new ProviderParameter{Key="resourcegroupname",Name="Resource Group Name", IsRequired=true , IsPassword=false},
+                    new ProviderParameter{Key="propdelay", Name="Propegation Delay", IsRequired=true, Value="60" }
```

Comment:
As mentioned, let's remove propagation delay from credentials. The less options users have to fill out the better.